zer0-os/zOS
# Remove emoji token parsing in favor of unicode inputs

### What does this do?

Removes the emoji token parsing when rendering a message.

### Why are we making this change?

https://www.notion.so/zerotech/9a4ed77a5ea14501a3e6ef7a33f5a546?v=21e3add7413040bfb3eaf77e8c689cec&p=f91ee19e9b5343729bd4acaf50f2ecce&pm=s

We've changed zero-web to store unicode characters rather than emoji tokens in messages. This is to align with how the mobile apps input emojis. This means there's no reason to continue to try to parse tokens.

### How do I test this?

Open messages in a channel and verify that they render. Old emoji tokens will appear as just the string. As the new features are deployed there will no longer be any tokens in new messages so this won't be an issue.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security? N/A
  1. How will this affect performance? N/A
  1. Does this change any APIs? N/A
